irohad/ametsuchi/impl/postgres_block_index.cpp

```diff
@@ -39,13 +37,20 @@ namespace {
     return (base % hash.hex() % height).str();
   }
 
+  std::string makeCommittedTxHashIndex(
```

Comment:
Please create a task and put a todo to rename this and similar methods. 

There are two issues:
1. We have nothing here to say "make"
2. We are not creating an index here